tests/test_api.py

```diff
@@ -655,9 +683,52 @@ def test_visualize_tangled_tree_layers(self, client, figure_type, data_model_jso
         assert response.status_code == 200
 
 
+@pytest.mark.schematic_api
+class TestValidationBenchmark():
+    @pytest.mark.parametrize('MockComponent_attribute', get_MockComponent_attribute())
+    def test_validation_performance(self, benchmark_data_model_jsonld, client, test_invalid_manifest, MockComponent_attribute ):
+        target_rows = 1000
+        endpoint_url = 'http://localhost:3001/v1/model/validate'
+
+        if MockComponent_attribute == 'Check Ages':
+            test_invalid_manifest[MockComponent_attribute][0]  = '6550'
+        elif MockComponent_attribute == 'Check Date':
+            test_invalid_manifest[MockComponent_attribute][0]  = 'October 21 2022'
+            test_invalid_manifest[MockComponent_attribute][2]  = 'October 21 2022'
+        elif MockComponent_attribute == 'Check Unique':
+            test_invalid_manifest[MockComponent_attribute][0]  = 'str2'
+
+
+        # Isolate single attribute of interest, keep `Component` column
+        single_attribute_manfiest = test_invalid_manifest[['Component', MockComponent_attribute]]
+
+        # Extend to ~1000 rows in size to for performance test
+        multi_factor = ceil(target_rows/single_attribute_manfiest.shape[0])
+        large_manfiest = pd.concat([single_attribute_manfiest]*multi_factor, ignore_index = True)
+        # Convert manfiest to JSON for api endpoint
+        manifest_json = large_manfiest.to_json(orient='records')
+        
+        
+        params = { 
+            "schema_url": benchmark_data_model_jsonld,
+            "json_str": manifest_json,
```

Comment:
Would it possible to continue using the json string until it's determined for sure that the parameter will be depreciated and we have a time estimate on when it will be?

Reply:
Let me double check with Anthony if he is still using it! I think it would be better to switch to use the csv option for testing once Anthony is moving away from using `json_str`

Reply:
@GiaJordan I just double checked with Anthony, and he is using a csv instead of a json_str. I think even if the option `json_str` is there, we should test based on what DCA is using?

Reply:
That makes sense! I'll switch to using a csv!